packages/mobile/src/screens/collection-screen/CollectionScreenDetailsTile.tsx

```diff
@@ -276,7 +276,9 @@ export const CollectionScreenDetailsTile = ({
       : !areAllTracksDeleted && (isQueued || (trackCount > 0 && !!firstTrack))
 
   const shouldShowPlay =
-    (isPlayable && hasStreamAccess) || doesUserHaveAccessToAnyTrack
+    !numericCollectionId ||
```

Comment:
maybe we rename this to "isSmartCollection"? but maybe that's not accurate because that includes audio-nft playlist? whatever you think.

Reply:
yeah I wasn't sure if that was semantically correct

Reply:
also this variable contains the numeric ID or null, so we'd need an additional bool for isSmartCollection

imo it's not super important right now, though it would be good to do a pass on smart collections in general